awx/ui_next/src/components/AppContainer/AppContainer.jsx

```diff
@@ -138,10 +138,12 @@ function AppContainer({ i18n, navRouteConfig = [], children }) {
     }
   }, [handleLogout, timeRemaining]);
 
+  const brandName = config?.license_info?.product_name || 'brand';
```

Comment:
`brand` was not selected to be translated. Is that the expected behavior?

Reply:
oh good catch!